### ISSUE TITLE
Weird tab behavior

### ISSUE THREAD

**Issue by jkarst** [Komodo/KomodoEdit#1100]
Hi there,

I've run into a problem using Komodo Edit, version 9.3.2, build 16460, platform win32-x86. 

Today, I switched tabs to a file I had been working on yesterday, and for some reason, all of my indents have switched to 8 characters.  

This file is a php class I had written from scratch using 4 character space indents.

See screenshot snippet:

[http://imgur.com/zm5SVzX](url)

Notice how the closing curly brace is at 4 character spacing. If I change the indent on it, it will align with 8 character spacing.

I've already done all the usual steps of double checking that spaces are preferred over tabs, that the indents are set to 4 characters, and that current file contents can't override my intentation settings (globally and for the current file).  I've restarted Komodo, closed and reopened the file.  All to no avail.

I can't repro this bug because it just seemed to happen for that one file.  My other open files are unaffected.

It's not the end of the world, but it is annoying.  Any ideas?

**Comment by mitchell-as** [comment]
Does the problem go away if you open a new tab (Ctrl+T) and select Troubleshooting > Reset File Preferences?

**Comment by jkarst** [comment]
So strange.

So the troubleshooting > reset file preferences didn't help.

But, neat:

   - I tried copy and pasting the entire contents from file (F1) from tab 1
   (T1) into another file (F2) in a second tab (T2) and saving it somewhere
   else.  The indentation works fine in F2.
   - I copy and pasted F2's contents back into F1 in the original T1, and
   the indent problem reappeared.
   - I then "save as"'s F1 as a different file (F3).  Indentation works
   fine as F3.
   - Then I saved F3 back as F1's original file name (all from within
    T1)...and indentation works fine again!  Nice.


I don't know if that's helpful to you, but it seemed to solve the problem
for me.

Before, it didn't matter if I closed and reopened a tab before loading
F1...the indent would just revert to 8 characters for some reason (new
indents, I mean).  Now when I close and reopen F1, everything works fine.
